hera_sim/tests/test_vis.py

```diff
@@ -92,26 +149,25 @@ def test_autocorr_flat_beam(uvdata, simulator):
     np.testing.assert_allclose(np.abs(v), np.mean(v), rtol=1e-3)
     np.testing.assert_almost_equal(np.abs(v), 0.5, 2)
 
-
-def test_viscpu_res_autocorr(uvdata):
-    I_sky = create_uniform_sky(nbase=5)
-    v = vis.VisCPU(
-        uvdata=uvdata,
-        sky_freqs=np.unique(uvdata.freq_array),
-        sky_intensity=I_sky,
-    ).simulate()
-
-    I_sky = create_uniform_sky(nbase=6)
-    v2 = vis.VisCPU(
-        uvdata=uvdata,
-        sky_freqs=np.unique(uvdata.freq_array),
-        sky_intensity=I_sky,
-    ).simulate()
-
-    # Ensure that increasing sky resolution smooths out
-    # any 'wiggles' in the auto-correlations of a flat sky.
-    assert np.std(np.abs(v)) >= np.std(np.abs(v2))
-
+### NEED TO CONSIDER WHETHER THIS TEST IS APPROPRIATE###
```

Comment:
If this tests behavior that we expect the simulator to have, then this is certainly a test we should have in place. If we can analytically place some constraints on the results of this test, then we should include those tests as well.